osu.Game/Overlays/WikiOverlay.cs

```diff
@@ -132,6 +132,24 @@ private void onSuccess(APIWikiPage response)
             }
         }
 
+        private void onFail()
+        {
+            LoadDisplay(new WikiMarkdownContainer
+            {
+                RelativeSizeAxes = Axes.X,
+                AutoSizeAxes = Axes.Y,
+                CurrentPath = $@"{api.WebsiteRootUrl}/wiki/",
+                Text = "There is something wrong when fetching this page. [Back to main page.](Main_Page)",
```

Comment:
Copy could use some work. Web uses this:

![2021-06-03-152727_433x64_scrot](https://user-images.githubusercontent.com/20418176/120652653-43f63900-c480-11eb-824f-3ce3278ad0ad.png)

but search isn't available, so I'd probably go with

```suggestion
                Text = $"Something went wrong when trying to fetch page \"{path.Value}\".\n\n[Back to main page.](Main_Page)",
```

maybe?